remove the word "support" from Windows pages, add Windows Server 2019

acozine opened the pull request:
##### SUMMARY
The list of Windows OSs you can manage with Ansible was different on the Windows FAQ and Windows Setup pages. Also, we used the word "support" on these pages in a confusing way. This PR updates the list of Windows OSs Ansible can manage and makes it clear that the lists reflect Ansible's technical capability, not support options from Red Hat.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com
windows modules

acozine committed:
remove support from Windows pages, add Windows Server 2019

jborean93 commented on docs/docsite/rst/user_guide/windows_setup.rst:
```diff
@@ -313,8 +313,8 @@ options are:
-  over HTTPS. The downstream libraries that Ansible currently uses only support
-  passing the CBT with NTLM authentication. Using Kerberos with
+  over HTTPS. The downstream libraries that Ansible currently uses can only
+  pass the CBT with NTLM authentication. Using Kerberos with
```
NTLM or Kerberos authentication

acozine replied:
Is this still true? 

```
Using Kerberos with
  ``CbtHardeningLevel = Strict`` will result in a ``404`` error.
```

jborean93 replied:
Nope Kerberos now supports passing the CBT as well as NTLM, sorry should have mentioned that before.

acozine committed:
jborean feedback

jborean93 committed:
Removed CBT info

Removed the CBT note about what transports Ansible actually supports. We've worked with both NTLM and Kerberos for a while now.

jborean93 commented:
@acozine I just removed that whole entry around CBT is only supported by Ansible on NTLM and Kerberos. It's not needed as they are the only 2 ones that actually use it which is already mentioned above. The entry was originally there because we only worked with CBT if using NTLM but that is no longer the case.
